hivemind/p2p/p2p_daemon_bindings/control.py

```diff
@@ -255,6 +270,9 @@ async def call_unary_handler(self, peer_id: PeerID, proto: str, data: bytes) ->
             callUnary=call_unary_req,
         )
 
+        if req.ByteSize() > self.persistent_conn_max_msg_size:
+            raise P2PHandlerError(f"Message size exceeds set limit {self.persistent_conn_max_msg_size}")
```

Comment:
I would say that `P2PHandlerError` is a error of the __remote__ handler. Let's use `P2PDaemonError`.

```suggestion
            raise P2PDaemonError(f"Message size exceeds set limit {self.persistent_conn_max_msg_size}")
```